web/components/SessionSpeakers/SessionSpeakers.tsx

```diff
@@ -60,7 +62,7 @@ export const SessionSpeakers = ({
       {speaker2 && <Speaker speaker={speaker2} variant="nonDesktop" />}
       <Shape />
     </div>
-    <div className={styles.column2} aria-hidden={speaker2 ? null : 'true'}>
+    <div className={styles.column2} aria-hidden={!Boolean(speaker2)}>
```

Comment:
Hm, I was about to write that this is probably fine. But now I see this note: https://w3c.github.io/aria/#h-note-53

> At the time of this writing, `aria-hidden="false"` is known to work inconsistently in browsers. As future implementations improve, use caution and test thoroughly before relying on this approach.

So I would really prefer to keep the original behavior of not outputting the attribute at all when the speaker is absent. As far as I can tell, TS is appeased by replacing the original `null` with `undefined`:

```suggestion
    <div className={styles.column2} aria-hidden={speaker2 ? undefined : true}>
```